src/App.jsx

```diff
@@ -14,70 +15,49 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+export const App = () => {
+  const [value, setValue] = useState('Jam');
+
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {value ? `${value} is selected` : 'No goods selected'}
+        {value && (
+          <button
+            onClick={() => setValue(null)}
```

Comment:
Create the helper function for this logic and use it in the two places
```suggestion
            onClick={resetGood}
```